packages/uiweb/src/lib/components/space/SpaceWidget/WidgetHeader.tsx

```diff
@@ -0,0 +1,227 @@
+    <Container>
+      {!isLive && 
+        <Section>
+          <ProfileContainer>
+            <PfpContainer>
+              <Pfp src={tempImageUrl} alt="pfp" />
+            </PfpContainer>
+            <HostContainer>
+              <HostName>
+                Laris
+                <Host status="live">Host</Host>
+              </HostName>
+              <HostHandle status="live">
+                {/* Fetch the handle from Lenster */}@hbjbjhb
+              </HostHandle>
+            </HostContainer>
+          </ProfileContainer>
+          <Item display={'flex'} alignSelf={'flex-start'} alignItems={'center'}>
+            {isHost &&
+              <Button padding='6.5px 16.5px'>Edit space</Button>
+            } 
+            <Image
+              src={SettingsIcon}
+              alt="Settings icon"
+            />
+            <Item marginLeft={'8px'} display={'flex'}>
+              <Image
+                onClick={() => setIsMinimized(!isMinimized)}
+                src={isMinimized ? CaretUpIcon : CaretDownIcon}
+                alt="Maximize/Minimize icon"
+              />
+            </Item>
+            <Item marginLeft={'8px'} display={'flex'} onClick={handleCloseWidget}>
+              <CloseSvg stroke='white' height='15' width='15'/>
+            </Item>
+          </Item>
+        </Section>
+      }
+      <Section>
+        <Text fontSize={'16px'} fontWeight={700}>Lenster partners with Push Protocol to bring seamless and secure data transfer</Text>
+        {isLive &&
+          <Item display={'flex'} alignSelf={'flex-start'} alignItems={'center'} marginLeft={'24px'}>
+            <Image
+              src={SettingsIcon}
+              alt="Settings icon"
+            />
+            <Item marginLeft={'8px'} display={'flex'}>
+              <Image
+                onClick={() => setIsMinimized(!isMinimized)}
+                src={isMinimized ? CaretUpIcon : CaretDownIcon}
+                alt="Maximize/Minimize icon"
+              />
+            </Item>
+            <Item marginLeft={'8px'} display={'flex'} onClick={handleCloseWidget}>
+              <CloseSvg stroke='white' height='15' width='15'/>
+            </Item>
+          </Item>
+          }
+      </Section>
+      {!isLive &&
+        <Item display={'flex'} marginTop={'12px'} alignItems={'center'}>
+          <Image
+            src={CalendarIcon}
+            alt="Calendar Icon"
+          />
+          <Item marginLeft={'4px'} fontSize={'14px'} fontWeight={600}>{formatDate(Date.now())}</Item>
+        </Item>
+      }
+      {isLive &&
+        <Section marginTop='12px'>
+          <Item display={'flex'} alignItems={'center'}>
+            <Image
+              src={LiveIcon}
+              alt="Calendar Icon"
+            />
+            <Text fontSize={'14px'} fontWeight={600} marginLeft={'4px'}>Live</Text>
+          </Item>
+          <Item display={'flex'} alignItems={'center'}>
+            <Item>
+              {/* ToDo: Add participants icons */}
```

Comment:
Since the Live/Scheduled status is also being reused, we can have it on the resuables, and import it here, along with the participant container.